electrolyzer/supervisor.py

```diff
@@ -137,6 +141,21 @@ def update_stack_status(self):
                 self.waiting[i] = 0
                 self.stacks_waiting_vec[i] = 0
 
+    def initialize_plant_stacks(self):
+        # TODO: decide how many stacks should be turned on
+        stack_number = round(self.initial_power_kW / self.stack_rating_kW) + 1
```

Comment:
If `self.initial_power_kW < self.stack_min_power` then one stack will be turned on but it won't get any power on the next time step. I don't think this is a very big deal since the only effect will be one additional shutdown, but we may want to catch this case with an if else:
`if self.initial_power_kW < self.stack_min_power: stack_number = 0`
`else: stack_number = round(...) + 1`